packages/plugin-medium-zoom/src/components/MediumZoom.tsx

```diff
@@ -10,13 +10,21 @@ interface Props {
 
 export default function MediumZoom(props: Props) {
   const { pathname } = useLocation();
+  const dark = useDark();
   const { selector = '.rspress-doc img', options = {} } = props;
 
   useEffect(() => {
-    setTimeout(() => {
+    let zoom: Zoom | undefined;
+    const timeout = setTimeout(() => {
       const images = document.querySelectorAll(selector);
-      mediumZoom(images, options);
+      zoom = mediumZoom(images, { ...options, background: dark ? '#000' : '#fff' });
```

Comment:
I think it's better to use `var(--rp-c-bg)`, which has adapted dark mode

Reply:
fixed